gokart/file_processor.py

```diff
@@ -266,6 +267,22 @@ def dump(self, obj, file):
         dump_obj.to_feather(file.name)
 
 
+class IniFileProcessor(FileProcessor):
+
+    def format(self):
+        return None
+
+    def load(self, file):
+        config = configparser.ConfigParser()
```

Comment:
TIPS: you can use LugiConfigParser https://github.com/spotify/luigi/blob/e1d5161765fa505fe131a43608b5476db9c515b2/luigi/configuration/core.py#L29C12-L29C29